spec/units/tab_completion/matchers/anything_matcher_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+require 'gitsh/tab_completion/matchers/anything_matcher'
+
+describe Gitsh::TabCompletion::Matchers::AnythingMatcher do
+  describe '#match?' do
+    it 'returns true for all input' do
+      matcher = described_class.new(double(:env))
+
+      expect(matcher.match?('foo')).to be_truthy
+      expect(matcher.match?('--all')).to be_truthy
+      expect(matcher.match?('')).to be_truthy
+    end
+  end
+
+  describe '#completions' do
+    it 'returns an empty array' do
+      matcher = described_class.new(double(:env))
+
+      expect(matcher.completions('foo')).to eq([])
+      expect(matcher.completions('--all')).to eq([])
+      expect(matcher.completions('')).to eq([])
+    end
+  end
+
+  describe '#eql?' do
+    it 'returns true when given another instance of the same class' do
```

Comment:
I do in cases like this, where there would be one `it` per `context`, and the combined description is relatively short.